model/project_ref.go

```diff
@@ -2947,7 +2951,11 @@ func GetUpstreamProjectName(triggerID, triggerType string) (string, error) {
 
 // projectRefPipelineForValueIsBool is an aggregation pipeline to find projects that have the projectKey
 // explicitly set to the val, OR that default to the repo, which has the repoKey explicitly set to the val
+// Should not be used with project enabled field.
 func projectRefPipelineForValueIsBool(projectKey, repoKey string, val bool) []bson.M {
+	if projectKey == ProjectRefEnabledKey {
```

Comment:
i did remove all instances but I thought we might want a safeguard in case someone accidentally uses it but if the comment seems enough i can remove it!

Reply:
I think not having a RepoRefEnabledKey helps safeguard this!